doc/developer-notes.md

```diff
@@ -1041,8 +1041,15 @@ A few guidelines for introducing and reviewing new RPC interfaces:
 
   - *Rationale*: RPC methods registered with the same function pointer will be
     considered aliases and only the first method name will show up in the
-    `help` rpc command list.
+    `help` RPC command list.
 
   - *Exception*: Using RPC method aliases may be appropriate in cases where a
     new RPC is replacing a deprecated RPC, to avoid both RPCs confusingly
     showing up in the command list.
+
+- Use the `UNIX_EPOCH_TIME` constant when describing Unix epoch time or
+  timestamps in the RPC documentation.
+
+  - *Rationale*: User-facing consistency. Note that all times in Bitcoin are not
+    clocks (e.g. C++11 time points) but durations expressed in Unix epoch time
+    (e.g. in seconds since the epoch Jan 1 1970 GMT).
```

Comment:
(except that leap seconds are ignored, but I don't think this is worth mentioning here  :smile:)

Reply:
(credit to @MarcoFalke [here](http://www.erisian.com.au/bitcoin-core-dev/log-2019-11-26.html#l-368) for the "Note" part)

Reply:
On a second thought, while this is probably correct, it might be more confusing that helpful. A time_point is templated on clock and duration ( https://en.cppreference.com/w/cpp/chrono/time_point ). And assuming a C++20 system_clock, that duration is counted from unix time 0.

Reply:
So I'd suggest to just keep the rationale at "user-facing consistency"